packages/number-input/src/CustomSelectMenuButton/CustomSelectMenuButton.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import { ButtonProps } from '@leafygreen-ui/button';
+
+import { UnitSelectButton } from '../UnitSelectButton';
+
+/**
+ * Custom select menu unit button with a tooltip.
+ * Tooltip will show up if there is an ellipse.
+ *
+ * Passing down just the function which will be instantiated inside `Select`
+ * @internal
+ */
+export const CustomSelectMenuButton = React.forwardRef(
+  ({ children, ...props }: ButtonProps, forwardedRef) => (
+    <UnitSelectButton ref={forwardedRef} {...props}>
```

Comment:
Do we need to wrap `UnitButton`? Why can't we just pass it directly into `__INTERNAL__menuButtonSlot__`

Reply:
You're right; we don't need to wrap it.